best-practices/README.md

```diff
@@ -110,6 +110,10 @@ Testing
 
 * Avoid `any_instance` in rspec-mocks and mocha. Prefer [dependency injection].
 * Avoid `its`, `let`, `let!`, `specify`, and `before` in RSpec.
+  * Instead of using these, extract a method that you can pass the thing that
```

Comment:
I get a bit tripped up when reading this sentence, thoughts on `Instead of using using the above methods, extract a method that accepts the differing data as an argument` or something similar?